client/test/grammars/snaps/bitbake-conf.conf

```diff
@@ -0,0 +1,20 @@
+BBPATH .= ":${LAYERDIR}"
```

Comment:
This is a layer.conf example file. However, be advised that Yocto has other types of .conf files:

- local.conf
- bblayers.conf
- machine/*.conf
- distro/*.conf
which usually make use of .inc files in the same syntax.

Reply:
Do you think it is ok that I reuse the tests for bb files on `.inc` and `.conf` files?
I was wondering if I could reuse some of them since they share the grammar rules.

Also, I intend to make those changes in a different PR. Let this one be the grammar rules reviews

Reply:
Ack